core/src/main/resources/hudson/model/Messages.properties

```
@@ -113,7 +113,7 @@ ComputerSet.SlaveAlreadyExists=Agent called \u2018{0}\u2019 already exists
 ComputerSet.SpecifySlaveToCopy=Specify which agent to copy
 ComputerSet.DisplayName=Nodes
 
-Descriptor.From=(from <a href="{1}">{0}</a>)
+Descriptor.From=(from <a href="{1}" target="_blank">{0}</a>)
```

```suggestion
Descriptor.From=(from <a href="{1}" rel="noopener noreferrer" target="_blank">{0}</a>)
```

See https://github.com/jenkinsci/jenkins/pull/4910